builtin/mainmenu/tab_local.lua

```diff
@@ -107,32 +112,81 @@ else
 	end
 end
 
+local function get_disabled_settings(game)
+	if not game then
+		game = current_game()
+	end
+	local gameconfig
+	if game ~= nil then
+		local gamepath = game.path
+		gameconfig = Settings(gamepath.."/game.conf")
+	end
+	local disabled_settings = {}
+	if gameconfig then
+		local disabled_settings_str = (gameconfig:get("disabled_settings") or ""):split()
+		for _, value in pairs(disabled_settings_str) do
+			local state = false
+			value = value:trim()
+			if string.sub(value, 1, 1) == "!" then
+				state = true
+				value = string.sub(value, 2)
+			end
+			if valid_disabled_settings[value] then
+				disabled_settings[value] = state
+			else
+				core.log("error", "Invalid disabled setting in game.conf: "..tostring(value))
+			end
+		end
+	end
+	return disabled_settings
+end
+
 local function get_formspec(tabview, name, tabdata)
 	local retval = ""
 
 	local index = filterlist.get_current_index(menudata.worldlist,
 				tonumber(core.settings:get("mainmenu_last_selected_world"))
 				)
+	local disabled_settings = get_disabled_settings()
```

Comment:
This needs to get the game id from the selected world, and then use that to get the disabled settings. On Android, the game bar is not available meaning that the current game will always be nil